submit.go

```diff
@@ -665,46 +687,47 @@ func buildReportBody(p payload, newline, quoteChar string) *bytes.Buffer {
 	return &bodyBuf
 }
 
-func buildGenericIssueRequest(p payload, listingURL string) (title, body string) {
-	bodyBuf := buildReportBody(p, "  \n", "`")
```

Comment:
`buildReportBody` should be removed, but I see it's still being used for sending email notifications. Was there any reason not to use the template for emails as well?

Reply:
Well, it opens up a can of worms. Should we use the same template for both notifications? If so, the template has to handle the quote chars. If not, you've got two templates to keep in sync.

That said, keeping two templates in sync is arguably easier than keeping a template and a hard-coded impl in sync, so maybe that answers my question...